### PR TITLE
fix(Circle): missing bounding box

### PR DESCRIPTION
**Pull request by BotellaA**

### CODE REVIEW

File: src/geode/geometry/basic_objects/circle.cpp [hunk 1]
```diff
@@ -67,6 +69,33 @@ namespace geode
...
+        const auto angle = [this]( Vector3D direction ) {
+            double dP = plane_.normal().dot( direction );
+            if( dP <= -1.0 )
+            {
+                return M_PI;
+            }
+            if( dP >= 1.0 )
+            {
+                return 0.;
+            }
```
Comment (panquez) [inline src/geode/geometry/basic_objects/circle.cpp:84]: if direction and normal are normalized, is this possible / needed?

File: src/geode/geometry/basic_objects/circle.cpp [hunk 2]
```diff
@@ -67,6 +69,33 @@ namespace geode
...
+
+            return std::acos( dP );
+        };
+        const auto angle_x = angle( { { 1, 0, 0 } } );
+        const auto angle_y = angle( { { 0, 1, 0 } } );
+        const auto angle_z = angle( { { 0, 0, 1 } } );
+        const auto translation = Vector3D{
+            { std::sin( angle_x ), std::sin( angle_y ), std::sin( angle_z ) }
```
Comment (panquez) [inline src/geode/geometry/basic_objects/circle.cpp:92]: sin(arccosx) = sqrt(1-x^2)

**Review by panquez — APPROVED** [review]
missing python bindings and tests

File: src/geode/geometry/basic_objects/circle.cpp (hunk 1 above)
Comment (BotellaA) [reply src/geode/geometry/basic_objects/circle.cpp:84]: it only works if they are normalized but it is the case here

**Pull request merged**